.github/workflows/pr_label_size.yml

```
@@ -0,0 +1,33 @@
+name: labeler
+
+on: [pull_request]
+
+jobs:
+    labeler:
+      permissions:
+        pull-requests: write
+        contents: read
+        issues: write
+      runs-on: ubuntu-latest
+      name: Label the PR size
+      steps:
+        - uses: codelytv/pr-size-labeler@v1
+          with:
+            xs_label: 'size/xs'
```

Consider using `:` instead of `/` (or another delimiter). IMO it's more readable.
Also, consider using capital letters
```suggestion
            xs_label: 'size:XS'
```